.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "Java",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": ".."
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/java:1": {
+			"version": "17",
```

Comment:
TLC’s minimum Java requirement is Java 11.

Reply:
done